app/models/course/assessment/programming_evaluation.rb

```diff
@@ -0,0 +1,62 @@
+# Represents one pending programming question evaluation.
+#
+# This is different from a job in that jobs are assigned to an arbitrary worker. These
+# programming question evaluations can only be run on a specific worker,
+#  a) has permission to access the course
+#  b) has the programming language runtime to evaluate this submission
+#
+# Like trackable jobs, elements in this table are ephemeral.
+class Course::Assessment::ProgrammingEvaluation < ActiveRecord::Base
```

Comment:
No job. This looks like a job, but does not involve any ActiveJob, because it runs on the evaluator and not on our job workers.